example.php

```diff
@@ -5,8 +5,8 @@
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
 
-require_once('src/jsonRPCClient.php');
 require_once('src/daemonRPC.php');
```

Comment:
If we use the `composer` installation, we only use the `require_once 'vendor/autoload.php'`.